layouts/_default/baseof.header-critical.css

```diff
@@ -6,61 +6,103 @@ input[type="checkbox"][id|="open"] {
 
 .announcement {
   width: 100%;
-  background-color: #f2f2f5;
-  padding: 6px 20px;
-  box-sizing: border-box;
+  background-color: var(--color-fg);
+  color: white;
   text-align: center;
-  font-size: 0.875rem;
+  font-size: var(--text-font-size-sm);
+  padding: 0.5rem 0;
+}
+
+.announcement .announcement--content {
+  padding: var(--block-padding);
+  margin: auto;
+  max-width: var(--block-width);
 }
 
 .announcement a {
-  color: inherit;
+  color: white;
   text-decoration: none;
 }
 
+.announcement a:hover {
+  color: var(--color-reima-red);
+}
+
 body > header {
   position: sticky;
   top: 0;
-  border-bottom: 1px solid #bfbfbf;
-  padding: 10px 0;
-  background: #fff;
+  border-bottom: var(--border);
+  padding: 0;
+  background: white;
   z-index: 1;
 }
 
 .menu {
   display: flex;
   align-items: center;
+  height: var(--menu-height);
 }
 
-.menu input:focus + label.menu__hamburger {
+.menu input:focus + label.menu__hamburger,
+.menu
+  input:focus
+  + label
+  + .overlay
+  > .overlay__inner
+  > .overlay__header
+  > label {
```

Comment:
There has to be an easier selector than this. Or?

Reply:
I can take overlay__inner out then it would look a bit nicer and still works fine on current menu.
.menu input:focus + label + .overlay .overlay__header > label